index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+
+    <head>
+        <nav>
+            <h3 id="logo">My Logo</h3>
+            <img src="./images/Enabled.svg" alt="Menu" id="menu">
+        </nav>
+    </head>
+    <main>
+        <div class="headline">
+            <img src="./images/header-shapes mobile.svg" alt="background" id="head-background">
+            <div class="intro">
+                <h1>I’m Soban. Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                    contact me.</p>
+                <p id="connect">LET'S CONNECT</p>
+                <div class="social-links">
+                    <a href="https://twitter.com/SobanSyeed"><img src="./images/Vector.svg" alt="Twitter"></a>
+                    <a href="https://www.linkedin.com/in/soban-syeed-82985b241/"><img src="./images/Linkedin icon.svg"
+                            alt="LinkedIn"></a>
+                    <a href=""><img src="./images/Vector (2).svg" alt=""></a>
+                    <a href="https://github.com/SnakyMz"><img src="./images/Vector (1).svg" alt="Github"></a>
+                    <a href=""><img src="./images/Vector (3).svg" alt=""></a>
+                </div>
```

Comment:
- [ ] I think your social media links should be a list of icons a use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href Like this
  ```html
   <ul>
        <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
        <!-- other links here -->
   </ul>

Reply:
Wow that would actually make it more semantic. Thanks for the suggestion. As you see I have not included the href for two icons ('M' and 'victory sign'), as I searched for it but couldn't find the website names or location. So can I remove them or just leave them be without the hrefs?